tests/JustSaying.UnitTests/JustSaying.UnitTests.csproj

```diff
@@ -12,8 +12,10 @@
   <ItemGroup>
     <PackageReference Include="AWSSDK.SimpleNotificationService" Version="$(AwsSdkSnsVersion)" />
     <PackageReference Include="AWSSDK.SQS" Version="$(AwsSdkSqsVersion)" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.0" />
```

Comment:
I think you can remove this as it's a transient dependency from `JustSaying.csproj`.